package.json

```diff
@@ -0,0 +1,82 @@
+{
+  "name": "@ssv/ngx.tilt",
+  "version": "0.1.0",
+  "description": "Angular Wrapper for Vanilla Tilt a smooth 3D tilt.",
+  "keywords": [
+    "ngx",
+    "angular2",
+    "angular",
+    "tilt",
+    "ux",
+    "ui"
+  ],
+  "scripts": {
+    "start": "npm run build -- --watch",
+    "test": "jest",
+    "tdd": "jest --watch",
+    "test-debug": "node --inspect-brk ./node_modules/jest/bin/jest.js --runInBand --env jest-environment-node-debug",
+    "test-devt": "devtool ./node_modules/jest/bin/jest.js --runInBand --env jest-environment-node-debug",
+    "clean": "rimraf dist",
+    "lint": "tslint -p . -c tslint.json -t stylish",
+    "build": "ng-packagr -p ng-package.json",
+    "prepare-release": "gulp prepare-release",
+    "publish:rel": "npm publish dist --access=public"
+  },
+  "author": "Kurt Aquilina <kaquilina92@gmail.com>",
+  "license": "MIT",
+  "private": false,
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ssv/ngx.tilt"
+  },
+  "peerDependencies": {
+    "@angular/core": ">=6.0.0",
+    "@types/vanilla-tilt": "^1.4.1",
```

Comment:
i dont think this is needed no? if so remove from readme installs aswell

Reply:
it is since if you need types for example `TiltOptions` unless we re-export